core/src/test/scala/unit/kafka/server/ListOffsetsRequestTest.scala

```diff
@@ -55,7 +55,7 @@ class ListOffsetsRequestTest extends BaseRequestTest {
       .build()
 
     val debugReplicaRequest = ListOffsetsRequest.Builder
-      .forReplica(ApiKeys.LIST_OFFSETS.latestVersion, ListOffsetsRequest.DEBUGGING_REPLICA_ID)
+      .forReplica(ApiKeys.LIST_OFFSETS.latestVersion(), ListOffsetsRequest.DEBUGGING_REPLICA_ID)
```

Comment:
Why do we need to add the brackets?

Reply:
We do not, I will raise another version tomorrow morning which address this + fix the merge conflicts

Reply:
Corrected